src/renderer/components/Input.vue

```diff
@@ -148,12 +148,15 @@ export default {
   &.has-error {
     border-color: $red;
   }
+
+  &.disabled {
+    opacity: .8;
+    pointer-events: none;
+  }
```

Comment:
While we appreciate the effort, we defer to our UX Designer for changes like these.